src/controllers/notifications.controllers.js

```diff
@@ -0,0 +1,46 @@
+import Notifications from "../models/Notifications.js";
+
+// Get notifications by user
+export const getNotificationsByUserId = async (req, res) => {
+  try {
+    const userId = req.params.user_id;
+    const response = await Notifications.find({ user_id: userId });
+
+    if (!response || response.length === 0) {
```

Comment:
let's remove this validation, if there are no notifications, then we should return an empty array 

res.status(200).json({ data: response ?? [], error: null });